app/tests/spec/views/sign_in.js

```diff
@@ -213,6 +213,23 @@ function (chai, $, sinon, p, View, Session, AuthErrors, Metrics, FxaClient,
           });
       });
 
+      it('shows error message to locked out users', function () {
+        sinon.stub(view.fxaClient, 'signIn', function () {
+          return p.reject(AuthErrors.toError('ACCOUNT_LOCKED'));
+        });
+
+        var password = 'password';
+        $('[type=email]').val(email);
+        $('[type=password]').val(password);
+        return view.submit()
+          .then(function () {
+            assert.isTrue(view.isErrorVisible());
+            assert.include(view.$('.error').text().toLowerCase(), 'locked');
+            assert.isTrue(TestHelpers.isEventLogged(
+                metrics, 'error.signin.auth.121'));
```

Comment:
[metrics.errorToId](https://github.com/mozilla/fxa-content-server/blob/bce776bbc2a1d6852e9f7d991d770a8ffb40a045/app/scripts/lib/metrics.js#L242) exists to create these error strings. Using that would be less brittle than manually creating the strings.